Network flag not being recognized

I recently bumped to version beta.6 and now I'm receiving this error when running
 heartbeat:

```bash
heartbeat_1  | 2021-07-01T11:01:24.610Z INFO    browser/browser.go:39   Synthetic monitor detected! Please note synthetic monitors are an experimental unsupported feature!
heartbeat_1  | 2021-07-01T11:01:24.986Z INFO    synthexec/synthexec.go:128      Running command: /tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3 in directory: '/tmp/elastic-synthetics-391316926'
heartbeat_1  | 2021-07-01T11:01:25.247Z INFO    synthexec/synthexec.go:224      stderr: error: unknown option '--network'
heartbeat_1  | 2021-07-01T11:01:25.250Z ERROR   synthexec/synthexec.go:172      Error waiting for command /tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3: exit status 1
heartbeat_1  | 2021-07-01T11:01:25.250Z INFO    synthexec/synthexec.go:176      Command has completed(1): /tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3
heartbeat_1  | 2021-07-01T11:01:25.250Z WARN    synthexec/synthexec.go:183      Error executing command '/tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3': exit status 1
```

_Originally posted by @picolloo in https://github.com/elastic/synthetics/issues/332#issuecomment-872145333_

--- a/__tests__/cli.test.ts
+++ b/__tests__/cli.test.ts
@@ -58,7 +58,23 @@ describe('CLI', () => {
     });
   });
 
-  it('mimick heartbeat with `--rich-events` flag', async () => {
+  it('mimick 7.13 heartbeat wiwth all flags', async () => {
+    const cli = new CLIMock([
+      join(FIXTURES_DIR, 'example.journey.ts'),
+      '--network',
+      '--json',
+      '--screenshots',
+      '--suite-params',
+      '{}',
+      '--outfd',
+      process.stdout.fd.toString(),
+    ]);
+    await cli.waitFor('journey/network_info');
+    expect(cli.output()).toBeDefined();
+    expect(await cli.exitCode).toBe(0);
+  });
+
+  it('mimick new heartbeat with `--rich-events` flag', async () => {
     const cli = new CLIMock([
       join(FIXTURES_DIR, 'fake.journey.ts'),
       '--rich-events',

--- a/__tests__/e2e/scripts/setup.sh
+++ b/__tests__/e2e/scripts/setup.sh
@@ -33,4 +33,4 @@ echo "" # newline
 echo "${bold}Starting elasticsearch , kibana and synthetics docker${normal}"
 echo "" # newline
 
-STACK_VERSION=8.0.0-SNAPSHOT docker-compose --file docker-compose.yml up --remove-orphans > ${TMP_DIR}/docker-logs.log 2>&1 &
+STACK_VERSION=7.13.1 docker-compose --file docker-compose.yml up --remove-orphans > ${TMP_DIR}/docker-logs.log 2>&1 &

--- a/src/parse_args.ts
+++ b/src/parse_args.ts
@@ -63,6 +63,7 @@ program
       .choices(['on', 'off', 'only-on-failure'])
       .default('on')
   )
+  .option('--network', 'capture network information for all journeys')
   .option(
     '--dry-run',
     "don't actually execute anything, report only registered journeys"

